src/test/java/api/support/fakes/FakeOkapi.java

```diff
@@ -463,6 +477,16 @@ private void registerCirculationRulesStorage(Router router) {
     });
   }
 
+  private void registerOpeningHours(Router router) {
+    router.get("/calendar/periods")
+      .handler(routingContext -> {
+        routingContext.response()
+          .setStatusCode(200)
+          .putHeader("content-type", "application/json")
+          .end(OpeningPeriodsExamples.oneDayPeriod().create().toString());
+      });
+  }
```

Comment:
Why is this needed? Have opening hours not been used by circulation before?

Reply:
Yes, other endpoints like /calendar/periods/:id/period return different entities with different structure. OpeningPeriods haven't been used in circulation before and now they are used by OverduePeriodCalculatorService (CIRC-548), which I'm implementing API tests for in this PR.

Reply:
Please could it be organised with the other calendar related endpoints, rather than directly in `FakeOkapi`?

Reply:
Actually, other calendar endpoints ("/calendar/periods/:id/period", "/calendar/periods/:id/calculateopening") are handled in FakeOkapi as well - just after this one.